src/components/CreateResetPassword/components/PasswordRequirementsList/PasswordRequirementsList.js

```diff
@@ -0,0 +1,94 @@
+import React, {
+  useState,
+  useEffect,
+} from 'react';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { Layout } from '@folio/stripes-components';
+
+import usePasswordRules from './queries';
+
+import styles from './PasswordRequirementsList.css';
+
+const passwordRequirementsNames = [
+  'password_length',
+  'numeric_symbol',
+  'special_character',
+  'alphabetical_letters',
+];
+
+const PasswordRequirementsList = ({ passwordValue }) => {
+  const intl = useIntl();
+
+  const [requiredRules, setRequiredRules] = useState([]);
+  const [unfulfilledRules, setUnfulfilledRules] = useState([]);
+
+  const { rules } = usePasswordRules();
+
+  useEffect(() => {
+    if (requiredRules.length <= 0 && rules?.length > 0) {
+      const requiredRulesSet = rules
+        .filter(rule => passwordRequirementsNames.includes(rule.name))
+        .map(rule => {
+          const splittedRuleDescription = rule.description.split(' must ');
+
+          rule.description = intl.formatMessage(
+            { id: 'stripes-core.createResetPassword.ruleTemplate' },
+            { description: splittedRuleDescription[1] },
```

Comment:
It would be really helpful to have some comments in here explaining what's going on. Without an example, splitting on `must` and passing off a substring as the description is totally opaque. I think rule descriptions tend to look like 

> The password must blah blah blah

and this just stripes off "The password must". So, pretty simple, but not that obvious.